playwright.config.js

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from "@playwright/test";
+
+export default defineConfig({
+  webServer: {
+    command: "npx vite --port 8080",
```

Comment:
This hardcodes the port again.. if we use playwright test runner to manage the webserver, then I'm not sure how to tell it to autopick port 0 but then figure out what to listen for. Think this is OK for now? Using the test runner to manage the server seems much nicer than each test manually doing it

Reply:
I think this is fine for now